zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -152,6 +152,12 @@ object ServerSpec extends HttpRunnableSpec(8088) {
       testM("200 response with encoded path") {
         val actual = status(!! / "get%2Fsuccess")
         assertM(actual)(equalTo(Status.OK))
+      } +
+      testM("Multiple 200 response") {
+        val data = Gen.listOfN(5)(Gen.fromEffect(status(!! / "success").orElse(UIO(Status.INTERNAL_SERVER_ERROR))))
```

Comment:
- I don't think we need to use a `Gen` here and use the `repeat` operator on ZIO.
- We should consider larger numbers (eg 1024) for such use-cases. Essentially something that is larger than number of `threads` or `ports` etc.